openstack_dashboard/dashboards/project/images/templates/images/images/_detail_overview.html

```diff
@@ -32,6 +32,8 @@
     {% else %}
       <dd>{% trans "Never updated" %}</dd>
     {% endif %}
+    <dt>{% trans "Official" %}</dt>
+    <dd>{{  image.project_supported|yesno|capfirst }}</dd>
```

Comment:
Does it work to add this image code inside this dd outside the template? Or maybe it would be easier to add an `{% if %} here with the img tag.
```
<img title="Chameleon Supported Appliance" style="float:right;" ng-if="image.project_supported" height="20px" src="https://www.chameleoncloud.org/static/images/logo-small.png" />
```

Reply:
Ah I see this is just for the detail view.